### DISCUSSION TITLE
How to update / upgrade?

### DISCUSSION THREAD

**Discussion by woutersamaey** [evertramos/nginx-proxy-automation#379] [Q&A]
Once running in production, how to update? Is it a matter of using `git pull` and then something else?
The README only mentions the fresh start script, but what comes after that fresh start?

Maybe a good idea to expand the README a little bit?

Thanks

**Comment by evertramos** [comment]
Please check 

https://github.com/evertramos/nginx-proxy-automation/blob/main/docs/upgrade-guide.md

**Reply by woutersamaey** [reply]
This covers updating from 0.4 to 0.5, but what about small changes, patches, etc?

Are we supposed to run "git pull" from time to time and restart the containers?

If so, which steps/commands would you recommend?

Just want to make sure we're not running any old versions as most people tend to launch and forget.
